extensions/cms/src/cmsResource/tree/treeProvider.ts

```diff
@@ -63,7 +63,12 @@ export class CmsResourceTreeProvider implements TreeDataProvider<TreeNode>, ICms
 			if (registeredCmsServers && registeredCmsServers.length > 0) {
 				this.isSystemInitialized = true;
 				// save the CMS Servers for future use
-				await this._appContext.apiWrapper.setConfiguration(registeredCmsServers);
+				let toSaveCmsServers = JSON.parse(JSON.stringify(registeredCmsServers));
+				toSaveCmsServers.forEach((server) => {
+					server.ownerUri = undefined,
+						server.connection.options.password = '';
```

Comment:
Also please add a comment explaining that we're stripping the password out on purpose since it's saved to disk

---
In reply to: [285771274](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r285771274) [](ancestors = 285771274)